pkg/report/title_stat.go

```diff
@@ -86,3 +78,67 @@ func (ts *titleStat) visit(cb func(int, ...string), titles ...string) {
 		ts.Nodes[title].visit(cb, append(titles, title)...)
 	}
 }
+
+func (ts *TitleStat) ToBytes() ([]byte, error) {
+	return json.MarshalIndent(ts, "", "\t")
```

Comment:
Nit: I think it can be `json.MarshalIndent(ts, "", "")` - we don't care about indentation here.